src/pages/channel/ui/channel-type-chip.tsx

```diff
@@ -0,0 +1,25 @@
+import { Chip } from '@heroui/chip';
+
+import { CHANNEL_TYPE_NAME_MAP } from '@/pages/channel/config/chips';
+import { ChannelTypeKey } from '@/pages/channel/config/type';
+
+interface Props<T extends ChannelTypeKey> {
+  name: T;
+}
```

Comment:
```suggestion
interface Props {
  type: ChannelType;
}
```

1. `name` 대신 변수명을 `channelType`이나 `type`으로하면 더 가독성이 높을 것 같습니다!
2. 타입 지정을 `@/entities/channel`에 선언된 `ChannelType` 타입을 사용하면 `extends` 사용 안 하고 더 깔끔하게 작성할 수 있습니다~